### PR TITLE
ACPO inline model train bug fix

### PR DESCRIPTION
**Pull request by wangzm509**
1.Bypass some environment errors.
2.Fix some train errors.
3.Avoid the problem that the loss function doesn't match the task.
4.Provide .csv and config in table test for training inline model.
Now we can use config and csv(in table test) to train inline model.
cmd:
1.cd ACPO-model/src
2.python train.py --root-path . --user-config ../test/config/user_train_config.json --data-dir ./data --save-model --epochs 1 --task classification --algorithm nn --cv kfold --folds 2 --loss cross_entropy

### CODE REVIEW

**Review by tejas-srikanth — APPROVED** [review]
Changes look good to me. Might still want to wait for Tom and Amir's input though

**Comment by tsczajkowski** [comment]
Can you squash commits under your name into one? Most of the changes are related to the same topic, so perhaps keeping the commits separate is not as convenient.

**Comment by wangzm509** [comment]
> Can you squash commits under your name into one? Most of the changes are related to the same topic, so perhaps keeping the commits separate is not as convenient.

okey i will do that

File: ACPO-model/src/utils.py [hunk 1]
```diff
@@ -11,8 +11,12 @@
 plt.rcParams['agg.path.chunksize'] = 10000
 with warnings.catch_warnings():
     warnings.filterwarnings("ignore", category=FutureWarning)
-    import onnx
-    import onnx_tf.backend
+    '''
+    tensorflow stopped addon support a while ago,we are finding solution now
```
Comment (tsczajkowski) [inline ACPO-model/src/utils.py:15]: Space after ,

**Review by tsczajkowski — CHANGES_REQUESTED** [review]

File: ACPO-model/src/utils.py (hunk 1 above)
Comment (wangzm509) [reply ACPO-model/src/utils.py]: done

File: ACPO-model/src/losses.py [hunk 2]
```diff
@@ -2,11 +2,17 @@
...
+    exists_zero = torch.any(contains_zero)
+    assert exists_zero == False, "ERR:There are 0 in the target,can't use mse_norm loss"
     loss_per_sample = torch.nn.MSELoss(reduction='none')(output, target)
     loss_per_sample_normalized = loss_per_sample / target
     return torch.mean(loss_per_sample_normalized)
 
 def L1Loss_norm(output, target):
+    contains_zero = torch.eq(target, 0)
+    exists_zero = torch.any(contains_zero)
+    assert exists_zero == False, "ERR:There are 0 in the target,can't use mae_norm loss"
```
[the remark this answers is not in the document]
Comment (wangzm509) [reply ACPO-model/src/losses.py]: done

File: ACPO-model/src/losses.py [hunk 3]
```diff
@@ -2,11 +2,17 @@
 import torch.nn
 
 def MSELoss_norm(output, target):
+    contains_zero = torch.eq(target, 0)
+    exists_zero = torch.any(contains_zero)
+    assert exists_zero == False, "ERR:There are 0 in the target,can't use mse_norm loss"
```
[the remark this answers is not in the document]
Comment (wangzm509) [reply ACPO-model/src/losses.py]: done

**Review by tsczajkowski — APPROVED** [review]
